pyface/qt/QtCore.py

```diff
@@ -33,6 +33,23 @@
     __version__ = QT_VERSION_STR
     __version_info__ = tuple(map(int, QT_VERSION_STR.split(".")))
 
+elif qt_api == "pyqt6":
+    from PyQt6.QtCore import *
+
+    from PyQt6.QtCore import pyqtProperty as Property
+    from PyQt6.QtCore import pyqtSignal as Signal
+    from PyQt6.QtCore import pyqtSlot as Slot
+    from PyQt6.Qt import QCoreApplication
+    from PyQt6.Qt import Qt
```

Comment:
These two imports fail for me with PyQt6 from PyPI, but pass if I replace `PyQt6.Qt` with `PyQt6.QtCore`:
```
>>> from PyQt6.QtCore import Qt
>>> from PyQt6.QtCore import QCoreApplication
>>> from PyQt6.Qt import Qt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'PyQt6.Qt'
```

Reply:
Hmm. I didn't have [PyQt6-Qt](https://pypi.org/project/PyQt6-Qt/) installed. But installing that package doesn't seem to help. `PyQt6.Qt` becomes importable, but still doesn't contain `Qt` or `QCoreApplication`.

```
>>> import PyQt6.Qt
>>> from PyQt6.Qt import Qt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Qt' from 'PyQt6.Qt' (unknown location)
>>> from PyQt6.Qt import QCoreApplication
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'QCoreApplication' from 'PyQt6.Qt' (unknown location)
```

I'm not quite sure what the purpose of the PyQt6-Qt package is: it doesn't seem to contain anything of interest:
```
>>> from PyQt6.Qt import *
>>> dir()
['__annotations__', '__builtins__', '__doc__', '__loader__', '__name__', '__package__', '__spec__']
```